assets/data/team.json

```diff
@@ -1,4 +1,38 @@
 [
+
+    {
+        "name":"John Doe",
+        "role":"admin/maintainer/contributor",
+        "profile_pic":"path/to/photos/johnDoe.jpg",
+        "email":"johndoe@example.com",
+        "github_link": "https://github.com/johndoe",
+        "linkedIn_link": "https://www.linkedin.com/in/johndoe/"
+    },
+    {
+        "name":"Vivek Shah",
+        "role":"contributor",
+        "profile_pic":"../assets/team/vivek.jpg",
+        "email":"vivekshah9969@gmail.com",
+        "github_link": "https://github.com/vivekshah1801",
+        "linkedIn_link": "https://www.linkedin.com/in/vivekshah1801/"
+    },
+    {
+        "name":"vipul krishna",
+        "role":"contributor",
+        "profile_pic":"../assets/team/vipul.jpg",
+        "email":"vipul07.mathuria@gmail.com",
+        "github_link":"https://github.com/vipul1999",
```

Comment:
not part of source. please update your fork.

Reply:
these lines have been updated.

Reply:
done